src/charts/line.js

```diff
@@ -787,12 +793,60 @@ define(function(require){
                 .enter()
                   .append('line')
                     .attr('class', 'extended-x-line')
-                    .attr('x1', (-xAxisPadding.left - 30))
+                    .attr('x1', (-xAxisPadding.left - verticalShift))
                     .attr('x2', chartWidth)
                     .attr('y1', height - margin.bottom - margin.top)
                     .attr('y2', height - margin.bottom - margin.top);
         }
 
+        /**
+         * Draws custom user-defined lines onto the chart
+         * @return void
+         */
+        function drawCustomLines(){
+            svg.select('.custom-lines-group')
+                .selectAll('line')
+                .remove();
+
+            let yValues = customLines.map(it => it.y);
+
+            let getColor = yValue => {
+                const definedColor = customLines.find(it => it.y === yValue).color;
+                if(definedColor) {
+                    return definedColor;
+                }
+
+                return 'grey';
+            }
+
+            //draw a horizontal line to extend x-axis till the edges
+            verticalLines = svg.select('.custom-lines-group')
+                .selectAll('line.custom-line')
+                .data(yValues)
+                .enter()
+                .append('line')
+                .attr('class', 'custom-line')
+                .attr('x1', (-xAxisPadding.left - verticalShift))
```

Comment:
I would prefer the line to start at the same spot the chart line starts. As I see it on the screenshot you provided, this starts on the y axis area, and it didn't feel right design-wise.

Thanks!